src/popup/pages/ImportAccount/index.js

```diff
@@ -124,15 +126,20 @@ class ImportAccount extends React.Component {
       <p className={"import-title"}>{content}</p>
     </div>)
   }
+  onSubmit = (event) => {
+    event.preventDefault();
+  }
   render() {
     return (
       <CustomView
         title={getLanguage('importAccount_1')}
         history={this.props.history}>
-        <div className="import-account-container">
-          {this.renderInput()}
-        </div>
-        {this.renderBottom()}
+        <form onSubmit={this.onSubmit} autoComplete="off">
+          <div className="import-account-container">
+            {this.renderInput()}
+          </div>
+          {this.renderBottom()}
```

Comment:
Shouldn't we move the actual submit code out from the button click handler, into the form submit handler?

Reply:
Oh, I see it now.